src/context/fetch.ts

```diff
@@ -26,20 +26,25 @@ export const augmentRequestInit = (requestInit: RequestInit): RequestInit => {
   }
 }
 
+export const gracefullyFetch = <ResponseType = any>(
+  input: string | MockedRequest,
+  requestInit: RequestInit = {},
+): Promise<ResponseType> => {
+  return gracefully(fetch(input, requestInit))
+}
+
 /**
  * Wrapper around the native `window.fetch()` function that performs
  * a request bypassing MSW. Requests performed using
  * this function will never be mocked.
  */
-export const fetch = <ResponseType = any>(
+export const fetch = (
   input: string | MockedRequest,
   requestInit: RequestInit = {},
-) => {
+): Promise<Response> => {
```

Comment:
Wonder if we can still use the given `ResponseType` generic with a regular `fetch` return statement 🤔

Reply:
I don't know if it makes sense. Do you have an example of usage ?

Reply:
My though was to let the end developer type-annotate the type of the response body they would receive. I suppose since there's no such possibility with the regular `fetch`, the `ctx.fetch` may drop it as well. One can use type casting for that purpose. What do you think?

Reply:
Yes it's right, it makes sense